### ISSUE TITLE
Sneak Spread modifier doesn't seem to work properly.

### ISSUE THREAD

**Issue by Llalpaca** [Unknown025/Flans-Mod-Plus#13]
I was messing around with the new crouching and sprinting spread modifiers, and I have found that they don't actually work? I set the modifiers to various different numbers, but it seems they don't do anything, or they work differently than I think they do. I'm not sure about sprinting, but I know that sneaking modifier doesn't seem to change the value from default (I tried setting SneakSpreadMultiplier to 0.01 and 1.0). Maybe I'm doing something wrong but I'm not sure.

**Comment by GoldSloth** [comment]
Hi, could you send me the gun config you used to test it, please?

**Comment by Llalpaca** [comment]
[44_Type14Nambu.txt](https://github.com/Unknown025/Flans-Mod-Plus/files/5320624/44_Type14Nambu.txt)
[44_30Cal.txt](https://github.com/Unknown025/Flans-Mod-Plus/files/5320625/44_30Cal.txt)

I used these two

**Comment by GoldSloth** [comment]
Thank you, I'll look into it when I'm next available

**Comment by GoldSloth** [comment]
I tried your configs out and they seemed to work? Sorry it's been so long since this issue was posted.

**Comment by Unknown025** [comment]
Assumed fixed, feel free to reopen if otherwise.

**Comment by Llalpaca** [comment]
I don't know what it is, but I still can't get those variables to work in my game, at least in version 1.5.6. In 1.5.7, I can't get accuracy values to function at all.

**Comment by Llalpaca** [comment]
Got accuracy values to work in 1.5.7, can confirm the SneakSpreadMultiplier doesnt work for me there either.

**Comment by GoldSloth** [comment]
Okay, I'll put it on the testing list

**Comment by GoldSloth** [comment]
Three years wow, sorry it's taken so long to finally figure out what was happening.. 
So firstly there was a recent error with spread in 1.57 but I've just fixed that.
What was more likely, is the documentation was wrong. It's `SneakSpreadModifier` not multiplier.. didn't help the variables were called that too. So I've now added both versions to the config reader so there's no confusion. The latest source should have the issue fixed.
